controllers/infrastructure.cluster.x-k8s.io/kopsmachinepool_controller.go

```diff
@@ -166,9 +154,15 @@ func (r *KopsMachinePoolReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 		return resultDefault, nil
 	}
 
-	igName, err := getInstanceGroupNameFromKopsMachinePool(kopsMachinePool)
-	if err != nil {
-		return resultError, err
+	igName := kopsMachinePool.Name
+
+	if _, ok := kopsMachinePool.Spec.KopsInstanceGroupSpec.NodeLabels["kops.k8s.io/instance-group-name"]; !ok {
+		return resultError, fmt.Errorf("instance group name missing from kops ig config")
+	}
+
+	// Patch kops IG node label to be the same as the KMP object
```

Comment:
How are we going to handle Flux behaviour with this patch? We may not need this

Reply:
The idea behind this is to ensure we will follow kops behavior of having the correct tags in AWS that references the InstanceGroup